include/deal.II/grid/reference_cell.h

```diff
@@ -324,6 +347,30 @@ namespace ReferenceCell
 
           return 0;
         }
+
+        /**
+         * Map an ExodusII vertex number to a deal.II vertex number.
+         */
+        virtual unsigned int
+        exodusii_vertex_to_deal_vertex(const unsigned int vertex_n) const
+        {
+          Assert(false, ExcNotImplemented());
+          (void)vertex_n;
+
+          return 0;
+        }
```

Comment:
@bangerth wanted to add these here since `GeometryInfo` has UCD renumbering tables. Having them here would also make writing future code that loads higher-order mesh data easier. I don't feel strongly about it but I think this is the right place for them for those reasons.